cfme/infrastructure/resource_pool.py

```diff
@@ -38,9 +33,12 @@ class ResourcePool(Pretty):
         If given a provider_key, it will navigate through ``Infrastructure/Providers`` instead
         of the direct path through ``Infrastructure/Resourcepool``.
     """
+    appliance = CurrentAppliance()
     pretty_attrs = ['name', 'provider_key']
 
-    def __init__(self, name=None, provider_key=None):
+    def __init__(self, name=None, provider_key=None, appliance=None):
+        self.appliance = appliance or CurrentAppliance()
```

Comment:
changed but as far as I'm aware CurrentAppliance just a wrapper for get_or_create_current_appliance()